<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Распознование подписи на изображении</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
font-family: Times New Roman;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>481</width>
      <height>671</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGraphicsView" name="originalImageView"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="processedImageView"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>671</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="choseImageLabel">
         <property name="text">
          <string>Выберите изображение:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chooseImageButton">
         <property name="text">
          <string>Выбрать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="findSignaturesButton">
         <property name="text">
          <string>Найти подписи</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="signaturesList"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>481</width>
      <height>671</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGraphicsView" name="secondOriginalImageView"/>
     </item>
     <item>
      <widget class="QGraphicsView" name="secondProcessedImageView"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>10</y>
      <width>161</width>
      <height>671</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="secondChoseImageLabel">
         <property name="text">
          <string>Выберите изображение:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="secondChooseImageButton">
         <property name="text">
          <string>Выбрать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="compareSignaturesButton">
         <property name="text">
          <string>Сравнить подписи</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="secondSignaturesList"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="resultTextBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>1311</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
